src/proto/mgmt/svc.proto

```diff
@@ -68,14 +68,41 @@ message JoinResp {
 }
 
 message LeaderQueryReq {
-	string system = 1;		// System name.
+	string system = 1;	// System name.
 }
 
 message LeaderQueryResp {
 	string currentLeader = 1;
 	repeated string replicas = 2;
 }
 
+// RASEvent describes a RAS event in the DAOS system.
+message RASEvent {
+	string name = 1;		// Unique identifier, 64-char.
+	string timestamp = 2;		// Fully qualified timestamp (us) incl timezone.
+	uint32 severity = 3;		// Event severity Fatal/Warning/Error/Info.
+	string msg = 4;			// Human readable message describing event.
+	uint32 rank = 5; 		// (optional) DAOS rank involved in event.
+	string hostname = 6;		// (optional) Hostname of node involved in event.
+	bytes data = 7;			// (optional) Instance data treated as blob.
+	uint32 eid = 8;			// Unique numeric event identifier.
+	uint32 type = 9;		// Event type StateChange/InfoOnly.
+	uint32 instance_idx = 10;	// (optional) DAOS harness instance index.
+}
+
+// ClusterEventReq wraps an event subtype e.g. RASEvent.
+message ClusterEventReq {
+	uint32 id = 1;		// Sequence identifier for cluster events.
```

Comment:
for dRPC I think we call sequence numbers "sequence". If we do that here, we can avoid confusing the sequence number vs. event ID.